server/channels/app/platform/web_conn.go

```diff
@@ -27,15 +27,16 @@ import (
 )
 
 const (
-	sendQueueSize          = 256
-	sendSlowWarn           = (sendQueueSize * 50) / 100
-	sendFullWarn           = (sendQueueSize * 95) / 100
-	writeWaitTime          = 30 * time.Second
-	pongWaitTime           = 100 * time.Second
-	pingInterval           = (pongWaitTime * 6) / 10
-	authCheckInterval      = 5 * time.Second
-	webConnMemberCacheTime = 1000 * 60 * 30 // 30 minutes
-	deadQueueSize          = 128            // Approximated from /proc/sys/net/core/wmem_default / 2048 (avg msg size)
+	sendQueueSize                  = 256
+	sendSlowWarn                   = (sendQueueSize * 50) / 100
+	sendFullWarn                   = (sendQueueSize * 95) / 100
+	writeWaitTime                  = 30 * time.Second
+	pongWaitTime                   = 100 * time.Second
+	pingInterval                   = (pongWaitTime * 6) / 10
+	authCheckInterval              = 5 * time.Second
+	webConnMemberCacheTime         = 1000 * 60 * 30 // 30 minutes
+	deadQueueSize                  = 128            // Approximated from /proc/sys/net/core/wmem_default / 2048 (avg msg size)
+	websocketSuppressWarnThreshold = time.Minute
```

Comment:
Wondering whether this should be configurable for debugging purposes?

Reply:
Hmm .. what do you have in mind for debugging? If it's a local environment, then the developer should be able to change the code themselves.

Or were you thinking there is a use-case to change this dynamically in production systems?

Reply:
@isacikgoz - I am going to merge this, but we can keep this discussion going.

Reply:
Sorry this slipped from my notifications (due to okta enforcement 🤦 ). I was thinking changing dynamically in production to have a granular observation on these logs.